package.json

```diff
@@ -242,7 +241,8 @@
     "kafka-avro": "^3.2.0",
     "node-rdkafka": "^3.1.0",
     "pg-native": "^3.1.0",
-    "prisma": "^5.18.0"
+    "prisma": "^5.18.0",
+    "superagent": "^10.0.0"
```

Comment:
I guess you need to skip the superagent test < Node v18?

Reply:
Yes, skipped the tests